src/FOM/UserBundle/Resources/translations/messages.de.xlf

```diff
@@ -594,6 +594,54 @@
         <source>User credentials have expired.</source>
         <target>Anmeldedaten sind abgelaufen.</target>
       </trans-unit>
+      <trans-unit id="fom.user.user.index.datatable.emptytable">
```

Comment:
* Translation definition  `fom.user.user.index.datatable.emptytable` should be `mb.manager.admin.user.no-results`
* Translation should be an YAML file, XLF isn't wellcome
* `datatable` as key name namespace isn't necessary. What if we change DataTables to another plugin? 
* `user.user` - really?
*  FOM Bundle will be splitted in several modules, so please no `fom` any more.

Take a look at current administration translations defined as an YAML:
https://github.com/mapbender/mapbender/blob/release/3.0.6/src/Mapbender/ManagerBundle/Resources/translations/messages.de.yml#L30

Please rethink and change translations.